docker/server/init-defaults.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env sh
+
+# define defaults
+CLICKHOUSE_DEFAULT_DB="${CLICKHOUSE_DEFAULT_DB:-}";
+CLICKHOUSE_DEFAULT_USER="${CLICKHOUSE_DEFAULT_USER:-}";
+CLICKHOUSE_DEFAULT_PASSWORD="${CLICKHOUSE_DEFAULT_PASSWORD:-secret}";
+
+# if default user is defined - create it
+if [ -n "$CLICKHOUSE_DEFAULT_USER" ]; then
+  cat <<EOT >> /etc/clickhouse-server/users.d/default-user.xml
+  <yandex>
+    <!-- Docs: <https://clickhouse.tech/docs/en/operations/settings/settings_users/> -->
+    <users>
+      <${CLICKHOUSE_DEFAULT_USER}>
```

Comment:
i would expect that CLICKHOUSE_DEFAULT_USER will be used INSTEAD of user `default` defined in `users.xml` (so you should remove default user by adding `<default remove="remove" />`

Reply:
Yes, you right, changes made